_data/utility.rb

```diff
@@ -0,0 +1,130 @@
+module Jekyll
```

Comment:
It looks like Federalist isn't picking up your custom plugin. I was able to get it to work on my local machine by moving this file to`_plugins/utility.rb` and then adding `plugins_dir: ./_plugins` in the `_config.yml` file